dashboard/app/views/home/_homepage.html.haml

```diff
@@ -42,3 +42,20 @@
       = link_to pd_workshop_admins_path do
         %h3
           Directory for Workshop Admins
+    - courses_facilitated = Pd::CourseFacilitator.where(facilitator: current_user).map(&:course)
+    -if not courses_facilitated.empty?
```

Comment:
nit: You could use Ruby [any?](https://apidock.com/ruby/v2_5_5/Array/any%3F) or Rails [present?](https://apidock.com/rails/Object/present%3F).